tests/frontend/callbacks/test_dataset_callbacks.py

```diff
@@ -119,11 +119,6 @@ def file_path():
             "2f72477a-f051-43ee-bf8b-0d8f47b5e0a7",
             UUID("2f72477a-f051-43ee-bf8b-0d8f47b5e0a7"),
         ),
-        (
-            DatasetIdentifiers.OWNER,
-            "Seksjon for dataplattform",
```

Comment:
We should keep this case, and just modify the given and expected data, it's important to have a test case for every metadata field.